src/characters/zombie.js

```diff
@@ -4,6 +4,7 @@ import { objectDefinition, functionDefinition, callDefinition } from '@/utility/
 import { displayCoordinates } from '@/utility/graphics'
 
 class Zombie extends Character {
+  shadowSize = 0.27
```

Comment:
Might want to document more specifically what this number comes from. It might seem obvious but when used in the long line below it makes it kind of hard to understand at first.